DDD.Functions/GetPrizeDraw.cs

```diff
@@ -26,7 +26,8 @@ public static class GetPrizeDraw
             var (conferenceFeedbackRepo, sessionFeedbackRepo) = await feedbackConfig.GetRepositoryAsync();
             var conferenceFeedback = await conferenceFeedbackRepo.GetAllAsync(conference.ConferenceInstance);
             var sessionFeedback = await sessionFeedbackRepo.GetAllAsync(conference.ConferenceInstance);
-
+            
+            // we might need to use the SessionId instead of SessionName here
```

Comment:
Yep, I think that's better than relying on name when we don't need to, just keep it backwards compatible.

Reply:
Kept as is but this comment maybe for future refactoring 👍